Marlin/src/sd/cardreader.cpp

```diff
@@ -557,6 +561,11 @@ void CardReader::checkautostart() {
 
   if (!isDetected()) initsd();
 
+  #if ENABLED(EEPROM_SETTINGS) && DISABLED(FLASH_EEPROM_EMULATION)
+    SERIAL_ECHOLNPGM("Loading settings from SD");
+    (void)settings.load();
+  #endif
```

Comment:
This also seems a very strange place to insert this. And redundant….

Reply:
isnt settings.load made on the line before.. in initsd();